### PR TITLE
2023.11 update for Withings

### PR DESCRIPTION
**Pull request by joostlek**
## Proposed change

Add a section about the new sensors and how they work with all the changes added in 2023.11.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/102589
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/withings.markdown [hunk 1]
```diff
@@ -57,3 +58,14 @@ Withings will validate (with HTTP HEAD) these requirements each time you save yo
 
 The integration will automatically detect if you can use webhooks. This will enable the integration only to update when there is new data.
 The binary sensor for sleep will only work if the integration can establish webhooks with Withings.
+
+## Available data
+
+The integration provides several entities, some of which dynamically enabled based on the data present.
```
Comment (c0ffeeca7) [inline source/_integrations/withings.markdown]:
```suggestion
The integration provides several entities, some of which are dynamically enabled if data is available.
```

File: source/_integrations/withings.markdown [hunk 2]
```diff
@@ -57,3 +58,14 @@ Withings will validate (with HTTP HEAD) these requirements each time you save yo
...
+
+For example, measurement senors like weight will only work when there has been data registered for it in the last 14 days. So if you start using a new device, for example something to measure your temperature or you manually update a value in the app, the sensor will automatically appear.
```
Comment (c0ffeeca7) [inline source/_integrations/withings.markdown]:
```suggestion
For example, measurement sensors like weight only work when data has been registered in the last 14 days. So if you start using a new device, for example, to measure your temperature or you manually update a value in the app, the sensor automatically appears.
```

File: source/_integrations/withings.markdown [hunk 3]
```diff
@@ -57,3 +58,14 @@ Withings will validate (with HTTP HEAD) these requirements each time you save yo
...
+
+Sleep sensors will only be created if the integration can find sleep data for you in the last day.
```
Comment (c0ffeeca7) [inline source/_integrations/withings.markdown]:
```suggestion
Sleep sensors are only created if the integration can find sleep data for you within the last day.
```

File: source/_integrations/withings.markdown [hunk 4]
```diff
@@ -57,3 +58,14 @@ Withings will validate (with HTTP HEAD) these requirements each time you save yo
...
+
+For the workout calendar and the workout and activity sensors, it's the same, but 14 days.
```
Comment (c0ffeeca7) [inline source/_integrations/withings.markdown]:
```suggestion
Workout calendar and the workout and activity sensors show if the latest available data point is no older than 14 days.
```

Comment (c0ffeeca7) [reply source/_integrations/withings.markdown]: just an idea. Feel free to ignore.

**Review by c0ffeeca7 — COMMENTED** [review]
tiny tweaks

File: source/_integrations/withings.markdown (hunk 4 above)
Comment (joostlek) [reply source/_integrations/withings.markdown]: but this is also covering the workout calendar, hmmm

**Review by c0ffeeca7 — DISMISSED** [review]
LGTM, @joostlek :+1:

File: source/_integrations/withings.markdown [hunk 5]
```diff
@@ -57,3 +58,14 @@ Withings will validate (with HTTP HEAD) these requirements each time you save yo
 
 The integration will automatically detect if you can use webhooks. This will enable the integration only to update when there is new data.
```
Comment (c0ffeeca7) [inline source/_integrations/withings.markdown]:
```suggestion
The integration automatically detects if you can use webhooks. This enables the integration only to update when there is new data.
```

**Review by c0ffeeca7 — APPROVED** [review]
LGTM, @joostlek :+1:

**Pull request merged**